tests/conformance_tests.rs

```diff
@@ -0,0 +1,94 @@
+            r#"(ion_1_1 "Timestamp Second" (text "2023-03-23T10:12:21Z") (denotes (Timestamp second 2023 3 23 (offset 0) 10 12 21))) "#,
+            r#"(ion_1_1 "Timestamp Fractional" (text "2023-03-23T10:12:21.23Z") (denotes (Timestamp fraction 2023 3 23 (offset 0) 10 12 21 23 -2))) "#,
+        ];
+        for test in tests {
+            Document::from_str(test)
+                .unwrap_or_else(|e| panic!("Failed to load document: <<{}>>\n{:?}", test, e))
+                .run()
+                .unwrap_or_else(|e| panic!("Test failed for simple doc: <<{}>>\n{:?}", test, e));
+            }
+    }
+
+    #[test]
+    fn test_encoding() {
+        let test: &str = r#"
+             (ion_1_1
+                 (encoding (macro_table (macro m () 1)))
+                 (text "(:m)")
+                 (produces 1)
+             )"#;
+        Document::from_str(test)
+            .unwrap_or_else(|e| panic!("Failed to load document:\n{:?}", e))
+            .run()
+            .unwrap_or_else(|e| panic!("Test failed: {:?}", e));
+    }
+
+    #[test]
+    fn test_simple_docs() {
+        let tests: &[&str] = &[
+            "(document (produces ))",
+            "(document (toplevel a) (produces a))",
+            "(document (text \"a\") (produces a))",
+            "(ion_1_0 (produces ))",
+            "(ion_1_1 (produces ))",
+            "(document (and (produces ) (produces )))",
+            "(document (text \"a\") (not (and (produces b) (produces c))))",
+            "(ion_1_1 (binary 0x60 0x61 0x01 0xEB 0x01) (produces 0 1 null.int))",
+            r#"(ion_1_0 (then (text "a") (produces a)))"#,
+            r#"(ion_1_1 (text "a") (text "b") (text "c") (produces a b c))"#,
+            r#"(ion_1_1 (text "\"Hello\" null.int false") (denotes (String "Hello") (Null int) (Bool false)))"#,
+            r#"(ion_1_1 (each 
+                             (text "0")
+                             (binary 0x60)
+                             (denotes (Int 0)))
+                        )"#,
+            r#"(document (ivm 1 2) (signals "Invalid Version"))"#,
+            r#"(ion_1_1 (text "2.3") (denotes (Decimal 23 -1)))"#,
+        ];
+        for test in tests {
+            println!("Testing: {}", test);
+            Document::from_str(test)
+                .unwrap_or_else(|e| panic!("Failed to load document: <<{}>>\n{:?}", test, e))
+                .run()
+                .unwrap_or_else(|e| panic!("Test failed for simple doc: <<{}>>\n{:?}", test, e)); 
+            }
+    }
+}
+
+mod ion_tests {
+    use super::*;
+
+    #[test_resources("ion-tests/conformance/null.ion")]
+    #[test_resources("ion-tests/conformance/core/typed_null.ion")]
+    #[test_resources("ion-tests/conformance/core/string_symbol.ion")]
+    #[test_resources("ion-tests/conformance/core/empty_document.ion")]
+    #[test_resources("ion-tests/conformance/core/toplevel_produces.ion")]
+    // #[test_resources("ion-tests/conformance/ivm.ion")]
```

Comment:
```suggestion
    // #[test_resources("ion-tests/conformance/ivm.ion")]
```
?

Reply:
Yea, I had commented this out because it currently doesn't pass due to the symbol syntax that's needed. Removing it for now.